devops/ansible/roles/girder/tasks/girder.yml

```diff
@@ -9,6 +9,7 @@
         - git
         - libffi-dev
         - build-essential
+        - libgif-dev
```

Comment:
Is this a new dependency?

Reply:
No, it should have been here.  We've just been lucky that we've only used the ansible role where it has already been installed.